docker/openldap/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2'
+services:
+  openldap:
+    image: osixia/openldap:latest
+    container_name: openldap
+    domainname: "example.org" # important: same as hostname
+    hostname: "openldap"
```

Comment:
`domainname` and `hostname` are supposed to align together, otherwise the `OpenLDAP` cannot be initialized.

Reply:
@RealChrisL I asked @mars-lan before I filled the PR. He suggested that I sent to metadata-ingestion directly instead of `contrib`. 
as far as the `example.org` and `openldap` are supposed to align together. Unfortunately it is the  original `docker-compose.yml` which doesn't work there so I have to make modification here. You can refer discussion for the original docker-compose.yml for detail. https://gist.github.com/thomasdarimont/d22a616a74b45964106461efb948df9c

Reply:
I mean I am happy to move to `contrib`.

Reply:
I did go through the `osixia/openldap`, nonetheless, verified against the `Red Hat Enterprise Linux Workstation release 7.6 (Maipo)` and `Docker version 18.09.3, build 774a1f4`, the PR docker-compose.yml will hang in `openldap        | Waiting for OpenLDAP to start...` which is not initialized.
While the `hostname` and `domainname` are aligned such as [osixia/docker-openldap](https://github.com/osixia/docker-openldap/blob/fa517c29889c8715fbcc90bd3d5f8042459c440a/example/docker-compose.yml), the sever can been brought up successfully.

Reply:
Have you tried it with macOS?  The issue I have seen is with macOS. Running the [osixia/docker-openldap](https://github.com/osixia/docker-openldap/blob/fa517c29889c8715fbcc90bd3d5f8042459c440a/example/docker-compose.yml) udner macOS will give the following issue `openldap exited with code 1` as seen by others and me.
```
phpldapadmin    | Link /container/service/:ssl-tools/assets/default-ca/default-ca.pem to /container/service/ldap-client/assets/certs/ldap-ca.crt
phpldapadmin    | *** Running /container/run/startup/phpldapadmin...
phpldapadmin    | Set apache2 http config...
phpldapadmin    | Bootstap phpLDAPadmin...
openldap exited with code 1
```

Reply:
Got it, if in this case, would you please add the prerequisites in the README.md as well as remove the misleading comment e.g. (# important: same as hostname), thanks!